docs/home/contribute/localization.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Localizing Kubernetes Documentation
+---
+
+We're happy to add localizations (l10n) of Kubernetes documentation to the website!
+
+Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+
+## Workflow  
+
+All l10n work must be stored and tracked within the [Kubernetes organization](https://github.com/kubernetes).
+
+#### Repository
+
+A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs/cn](https://github.com/kubernetes/kubernetes-docs-cn).
+
+**Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
+{: .note}
+
+#### Project
+
+Teams must track their overall progress with a [GitHub project](https://help.github.com/articles/creating-a-project-board/).
+
+Projects must include columns for:
+- pending work
+- work in progress
+- completed work
+
+#### Team function
+
+L10n teams must provide a single point of contact: the name and contact information of a person who can respond to or redirect questions or concerns.
+
+L10n teams must provide their own repository maintainers.
+
+All l10n work must be self-sustaining with the team's own resources.
+
+Wherever possible, every localized page must be approved by a reviewer from a different company than the translator.
+
+### Upstream contributions
+
+Upstream contributions are welcome and encouraged!
+
+For the sake of efficiency, limit upstream contributions to a single pull request per week, containing a single squashed commit.
+
+## Output
+
+All localizations must include the following documentation at a minimum:
```

Comment:
"at a minimum" for getting into the official kubernetes/website repo or just as a recommended practice?

Reply:
> "at a minimum" for getting into the official kubernetes/website repo

This. ☝️ 

@chenopis and I are still discussing the ideal workflow for l10n output. It seems likely that we'll need to set up an automated process for exporting output from the l10n repo to k/website. That means we'll need to agree on a directory structure in both repos and provide good documentation for how it all works.

There's nothing your team needs to do here yet; just raising visibility on the workflow questions. 🙇